packages/create-strapi-starter/utils/child-process.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const { execSync } = require('child_process');
+const execa = require('execa');
+const hasYarn = require('./has-yarn');
+
+const installArguments = ['install', '--production', '--no-optional'];
+/**
+ * @param  {string} path Path to directory (frontend, backend)
+ */
+function runInstall(path) {
+  if (hasYarn) {
+    return execa('yarn', installArguments, {
+      cwd: path,
+      stdin: 'ignore',
+    });
+  }
+
+  return execa('npm', installArguments, { cwd: path, stdin: 'ignore' });
+}
+
+const args = ['concurrently', 'wait-on'];
```

Comment:
we should define those statically in the package.json with static version so we make sure a new version doesn't break the setup. avoid running any yarn add

Reply:
You will also need to run a yarn install in the root  instead of the yarn add

Reply:
I put them in the `package.json` as dependencies.  As devDependencies I kept getting an error `command not found` after the install 🤷‍♂️.  Am I missing something or maybe I misunderstood your comment above?

> Need to add devDependencies with npm-run-all or concurrently

Reply:
you run yarn install with --production that's why :)

Reply:
🤦